plugins/dropdown/README.md

```diff
@@ -0,0 +1,98 @@
+# Builder.io example plugin
```

Comment:
Let's call this pugin the "dynamic dropdown" plugin (as you can already make dropdowns with text fields, but this one is specifically special because the options can be dynamic from an API)

so let's update the other names, like the foldername to `dynamic-dropdown` etc

open to other suggestions as well like `remote-dropdown` `api-dropdown` etc too

Reply:
Dynamic dropdown actually (IMHO) is the most accurate description